src/Blockcore.Indexer.Core/Sync/SyncTasks/BlockIndexer.cs

```diff
@@ -196,10 +204,10 @@ await mongoData.InputTable.Indexes
 
                }).ContinueWith(async task =>
                {
-                  log.LogDebug($"Creating indexes on {nameof(InputTable)}.{nameof(InputTable.Address)}");
+                  log.LogDebug($"Creating indexes on {nameof(UnspentOutputTable)}.{nameof(UnspentOutputTable.Address)}");
 
-                  await mongoData.InputTable.Indexes
-                     .CreateOneAsync(new CreateIndexModel<InputTable>(Builders<InputTable>
+                  await mongoData.UnspentOutputTable.Indexes
```

Comment:
Aren't you missing in index on the `BlockIndex` on the `UnspentOutputTable`

Reply:
It is being created with the creation of the table